sql/2021/performance/unused_css_histogram.sql

```diff
@@ -0,0 +1,21 @@
+#standardSQL
+# Histogram of unused CSS bytes per page
+
+SELECT
+  IF(unused_css_kbytes <= 500, CEIL(unused_css_kbytes / 10) * 10, 500) AS unused_css_kbytes,
+  COUNT(0) / SUM(COUNT(0)) OVER (PARTITION BY 0) AS pct,
+  COUNT(0) AS pages,
+  SUM(COUNT(0)) OVER (PARTITION BY 0) AS total,
+  --only interested in last max one as a 'surprising metric'
+  MAX(unused_css_kbytes) AS max_unused_css_kb
```

Comment:
Is this covered by the 100th percentile in the following query?

Reply:
No, you can see the sample output here: https://docs.google.com/spreadsheets/d/13xhPx6o2Nowz_3b3_5ojiF_mY3Lhs25auBKM6eqGZmo/edit#gid=161319858